src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,44 @@
+import "./cart.css";
+import { Link } from "react-router-dom";
+import { CartCard, CheckoutCard } from "../../components";
+import { useCartContext } from "../../context/cart-context";
+import { useEffect } from "react";
+
+const Cart = () => {
+  const { cartState } = useCartContext();
+
+  useEffect(() => {
+    document.title = `(${cartState.cart.length}) Cart | Saiyan Merch`;
```

Comment:
Suggestion: You can create a custom hook, but for the time being this is also fine